src/components/field-key/revoke.vue

```diff
@@ -93,64 +93,56 @@ export default {
     "title": "Odebrat přístup uživatele",
     "introduction": [
       "Opravdu chcete odebrat přístup uživateli aplikace {displayName}?",
-      "Stávající příspěvky od tohoto uživatele zůstanou, ale kdokoli se na tohoto uživatele obrátí, bude muset vytvořit nové, aby pokračoval ve stahování formulářů nebo odesílání příspěvků.",
-      "Tuto akci nelze vrátit zpět."
```

Comment:
The source message is "This action cannot be undone.", and it was moved out of this component and into en.json5 for use in other components. Because it was in an array before, I decided not to use `@transifexKey`, so it will need to be retranslated. Once it is, it will appear in cs.json.